cli/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'bio.terra.pfb.java-common-conventions'
+}
+
+version 'unspecified'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'info.picocli:picocli:4.7.4'
+}
+
+jar {
+    manifest {
+        attributes 'Main-Class': 'bio.terra.pfb.JavaPfbCommand'
+    }
+    archiveBaseName.set('java-pfb-cli')
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+    }
+}
+
+test {
+    useJUnitPlatform()
+}
+
+sonar {
+    properties {
+        property "sonar.projectKey", "DataBiosphere_java-pfb-cli"
```

Comment:
Great question -- I was hoping to use the [monorepo](https://docs.sonarcloud.io/advanced-setup/monorepo-support/) setup that's available for sonarcloud that I think should do what you're talking about. But, it's really hard to fiddle with settings because engineers don't have access to the admin settings on sonarcloud. Tom flipped the monorepo switch on the project, but as far as I could tell, nothing changed. The two projects still went into two separate sonarcloud projects. 

This technically works -- I have two steps in GHA that runs against the two projects. Sonarcloud comments twice in the PR, once for each project (unfortunately, the comment don't distinguish between the two projects). 

I think this is good enough for now, but I would definitely be interested in looking at the monorepo setup more -- this would require more time from infosec or more permissions in sonarcloud.